docs/RECOMMENDATION_SYSTEM_DESIGN.md

```
@@ -51,35 +51,50 @@ AI 엔진: OpenAI Embedding API
 
 ## 📊 데이터 흐름
 
-### Step 1: 콘텐츠 저장
+### Step 1: 폴더에 링크 저장
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)